fix: remove clunky sentence

naomi-lgbt opened the pull request:
Checklist:

<!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->

- [X] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [X] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [X] My pull request targets the `master` branch of freeCodeCamp.
- [X] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

<!--If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.-->

Closes #XXXXX

<!-- Feel free to add any additional description of changes below this line -->

This sentence is fairly unclear as it is in English, and does not come
across as a translation-friendly string. Also does not seem to add
value to the challenge information.

Signed-off-by: nhcarrigan <nhcarrigan@gmail.com>

cc/ @RandellDawson

Commit:
feat: remove clunky sentence

This sentence is fairly unclear as it is in English, and does not come
across as a translation-friendly string. Also does not seem to add
value to the challenge information.

Signed-off-by: nhcarrigan <nhcarrigan@gmail.com>

moT01 reviewed (commented):
The instructions say:

```
Assign the following three lines of text into the single variable myStr using escape sequences.

FirstLine
    \SecondLine
ThirdLine
```

The sentence being removed describes that it's a tab in front of `/SecondLine`. I wonder if users might try to use spaces there. Perhaps we should add something to the instructions saying that it's a tab.

Commit:
fix: add note about tabs

Proposed by Tom - adds a note that specifies the second line is
indented via a tab escape, not spaces.

Signed-off-by: nhcarrigan <nhcarrigan@gmail.com>

nhcarrigan commented:
I went ahead and added a note to clarify that.

RandellDawson reviewed (dismissed):
I think the addition of the note helps.  LGTM
